base/profile.jl

```diff
@@ -103,23 +103,22 @@ will be used.
 
 The keyword arguments can be any combination of:
 
- - `format` can be `:tree` (default) or `:flat`.
+ - `format` -- Determines how the backtraces are printed. Can be `:tree` (default) or `:flat`.
```

Comment:
This is perhaps not so good since all the arguments here determine how backtraces are printed...